src/dotnet/MsbuildProject.cs

```diff
@@ -205,7 +209,27 @@ private Project GetEvaluatedProject()
         {
             try
             {
-                return _projects.LoadProject(ProjectRootElement.FullPath);
+                Project project;
+                if (_interactive)
+                {
+                    // NuGet need this environment variable to call plugin dll
+                    Environment.SetEnvironmentVariable("DOTNET_HOST_PATH", new Muxer().MuxerPath);
```

Comment:
Why not test coverage?
1. When Console logger is set. The output is in Console. There is no good way to redirect console output during unit test. Since xunit run in parallel redirect console output will affect other tests.

2. No easy way to output message during msbuild eval. Need to build a whole msbuildsdk

3. even build a whole msbuildsdk, msbuild used during test does not have sdk props and targets. And there is no msbuild locator for netcore